src/core/public/utils/index.ts

```diff
@@ -6,5 +6,8 @@
  * Side Public License, v 1.
  */
 
-export { MountWrapper, mountReactNode } from './mount';
-export { KBN_LOAD_MARKS } from './consts';
+export {
+  MountWrapper,
+  mountReactNode,
+  KBN_LOAD_MARKS,
+} from '@kbn/core-mount-utils-browser-internal';
```

Comment:
I thought about removing the entire utils folder but then decided against it in order to keep the codeowners reviews to a minimum. These exports are also from an `internal` type package that isn't meant for public (non-core) consumption.

As far as I can tell, these are imported in  
https://github.com/elastic/kibana/blob/6b34c7182c69016f23a159f544ba9e3791f943f2/examples/search_examples/public/search/app.tsx#L37
@elastic/kibana-app-services

https://github.com/elastic/kibana/blob/6b34c7182c69016f23a159f544ba9e3791f943f2/src/plugins/vis_types/vislib/public/vis_controller.tsx#L13
@elastic/kibana-vis-editors

https://github.com/elastic/kibana/blob/6b34c7182c69016f23a159f544ba9e3791f943f2/x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/bulk_actions/use_bulk_actions.tsx#L16
@elastic/security-solution

and 
https://github.com/elastic/kibana/blob/6b34c7182c69016f23a159f544ba9e3791f943f2/src/plugins/controls/public/services/storybook/overlays.tsx#L19
@elastic/kibana-presentation

Should I rethink that since there are only 4 other teams involved at the moment?
It would allow us to reduce our core code surface area, well, at least in that we get rid of the `src/core/public/utils` folder.

Reply:
These utils were always meant to be internal, as Core is supposed to be publicly framework-agnostic. That's the reason why there are public equivalent of those utilities exposed from the `kibana_react` plugin.

We definitely want to avoid external consumers using them. But this can totally be done in a follow-up to avoid increasing this PR's size.